src/together/cli/api/files.py

```diff
@@ -32,12 +32,17 @@ def files(ctx: click.Context) -> None:
     default=FilePurpose.FineTune.value,
     help="Purpose of file upload. Acceptable values in enum `together.types.FilePurpose`. Defaults to `fine-tunes`.",
 )
-def upload(ctx: click.Context, file: pathlib.Path, purpose: str) -> None:
+@click.option(
```

Comment:
Are we planning on making it mandatory in the future?

Reply:
Good question, I don't have a strong opinion on that and decided to fall back on the previously implemented logic. To be honest, I can just measure the time it takes to check a 5GB jsonl/Parquet file; if it's not too long, we can always conduct this check

Reply:
the 5GB limit is only temporary. I have a fix in the pipeline that should enable much larger files.

Reply:
In that case, let's maybe leave the check optional? If it takes too long for large files, users can skip it, but they don't want to find out the file was incorrect only after finetuning starts